src/reducers/index.js

```diff
@@ -101,6 +101,7 @@ const combinedReducer = combineReducers({
   textLabels: createReducer(true, TOGGLE_TEXT_LABELS, 'textLabels'),
   theme: createReducer('dark', TOGGLE_THEME, 'theme'),
   orientation: createReducer('vertical', TOGGLE_ORIENTATION, 'orientation'),
+  view: createReducer('flowchart', SET_VIEW, 'view'),
```

Comment:
@jitu5 @Huongg , i had created this reducer to switch between views but I think @jitu5 has also created one so I dunno maybe we can remove this one.